website/src/app/[lang]/[region]/v2/(home)/(sections)/faq.tsx

```diff
@@ -1,25 +1,71 @@
 import { DefaultParams } from '@/app/[lang]/[region]';
 import { Translator } from '@socialincome/shared/src/utils/i18n';
 import { BaseContainer, Typography } from '@socialincome/ui';
-import { FontColor } from '@socialincome/ui/src/interfaces/color';
+import {
+	Accordion,
+	AccordionContent,
+	AccordionItem,
+	AccordionTrigger,
+} from '@socialincome/ui/src/components/accordion2';
+import Link from 'next/link';
+
+type FaqQuestion = {
+	prompt: string;
+	answer: string;
+	links: {
+		title: string;
+		href: string;
+	}[];
+};
 
 export async function Faq({ lang, region }: DefaultParams) {
 	const translator = await Translator.getInstance({
 		language: lang,
 		namespaces: ['website-home2'],
 	});
 
+	const questions = translator.t<FaqQuestion[]>('section-6.questions');
+
 	return (
 		<BaseContainer>
-			<Typography>{translator.t('section-6.title-1')}</Typography>
-			<div>
-				{translator.t<{ text: string; color?: FontColor }[]>('section-6.question-1').map((title, index) => (
-					<Typography as="span" key={index} color={title.color}>
-						{title.text}{' '}
-					</Typography>
+			<Typography size="2xl" className="mb-12 text-center">
+				{translator.t('section-6.title-1')}
+			</Typography>
+			<hr />
+			<Accordion type="single" collapsible className="mb-10 w-full">
+				{questions.map((question, index) => (
+					<AccordionItem key={index} value={`item-${index}`}>
+						<AccordionTrigger>
+							<Typography size="lg" className="text-left">
```

Comment:
Is the "text-left" required? Seems to me that this should be the default and part of the AccordionTrigger.